### PR TITLE
Fix configureGamePass

### PR DESCRIPTION
**Pull request by Regalijan**
- Use new endpoint
- Fix request dying because of non-string form values
- Don't tell Roblox it's a PNG when it isn't

Error handling could be a bit better but I'm saving that for #800

### CODE REVIEW

File: lib/games/configureGamePass.js [hunk 1]
```diff
@@ -26,29 +26,20 @@ exports.optional = ['description', 'price', 'icon', 'jar']
...
     const httpOpt = {
-      url: '//www.roblox.com/game-pass/update',
+      url: `//apis.roblox.com/game-passes/v1/game-passes/${gamePassId}/details`,
       options: {
         method: 'POST',
         jar,
         headers: {
           'X-CSRF-TOKEN': token,
-          'Content-Type': 'multipart/form-data; boundary=----WebKitFormBoundaryKMFaNaAn4j7XeMO'
+          'Content-Type': 'multipart/form-data; boundary=-----------------------------205723423134616457963583450846'
```
Comment (alanbixby) [inline lib/games/configureGamePass.js:36]: Does `boundary` need to be defined? I was under the impression this is automatically generated by `form-data`

Comment (Regalijan) [reply lib/games/configureGamePass.js:36]: Probably not, I just copied it from the example request you gave me (as the header was manually specified already)

**Review by Neztore — APPROVED** [review]
I'm happy but won't merge since I see there's a pending review from Alan

**Review by alanbixby — CHANGES_REQUESTED** [review]
This needs updated typings documented; the old response is a `Promise<GamePassResponse>`, but it appears the new endpoint would be a `Promise<void>`.

This should be documented in the local JSDocs comment, `typings/jsDocs.ts`, and `typings/index.d.ts`

**Review by alanbixby — APPROVED** [review]
Retracting comment about types - the response is manually returning data about the endpoint, so the return type is the same. No updates to types needed.

**Pull request merged**